lib/Service/LockService.php

```diff
@@ -156,6 +157,10 @@ public function getLockForNodeIds(array $nodeIds): array {
 			}
 		}
 
+		if (count($expiredLocks) > 0) {
+			$this->locksRequest->removeIds($expiredLocks);
+		}
+
```

Comment:
This is basically the batch remove that was in the `TODO` above before.
It's another performance tweak but not really related to the the addition of
```php
			$this->lockCache[$nodeId] = false;
```
in line 135 above, right?

Reply:
Yes, should probably have split that in a separate commit